examples/integrations/oci/vault-reactive/src/main/resources/application.yaml

```diff
@@ -14,9 +14,8 @@
 # limitations under the License.
 #
 
-# The values are read from
+# The values under oci.properties are read from
```

Comment:
Is this a typo? I don't see `oci.properties` below.

Reply:
There are, such as: 
`vault-ocid: "${oci.properties.vault-ocid}"`